lib/ansible/modules/systemd_service.py

```diff
@@ -47,13 +47,13 @@
             - Run daemon-reload before doing any other operations, to make sure systemd has read any changes.
             - When set to C(true), runs daemon-reload even if the module does not start or stop anything.
         type: bool
-        default: no
+        default: false
```

Comment:
This will not actually fix how the docs are rendered. Any `type: bool` displays the choices yes/no (regardless of `default`). Take a look at the find module's read_whole_file_parameter which has the same inconsistency: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html#parameter-read_whole_file
The documentation uses `default: false`: https://github.com/ansible/ansible/blob/stable-2.13/lib/ansible/modules/find.py#L61-L69

Since the docs are consistent about displaying yes/no as the choices for boolean options, we could also make the documentation consistent by using `yes` instead of `true` in the descriptions.

```yaml
            - When set to C(yes), runs daemon-reload even if the module does not start or stop anything.
```

 In general I think there should be examples using both though since we don't recommend one over the other.

I wonder if there would be a way to link to https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#truthiness from all module bool options (without manually doing it, of course).

Reply:
`ansible-doc` itself does not do that type of conversions and shows the examples and types as written